openpype/hosts/maya/plugins/load/load_reference.py

```diff
@@ -246,6 +253,11 @@ def _post_process_rig(self, name, namespace, context, options):
                 pre_create_data={"use_selection": True}
             )
 
+        nodes = self[:]
+        create_rig_animation_instance(
+            nodes, context, namespace, log=self.log
+        )
```

Comment:
This call is somewhat ok. However... the above logic in this method should be moved into that library function instead to replace the legacy create call instead.

Reply:
I agree, but I dont have time to resolve it. Welcome to dismiss this PR.